.github/ISSUE_TEMPLATE/support-invite-member-to-github-org.md

```diff
@@ -0,0 +1,14 @@
+---
+name: 'Support: Invite member to GitHub Org'
```

Comment:
I would rename the title to 'Invite member to consortium GitHub organization' to avoid confusion. 
I bet otherwise you could get the impression, that we can invite people to eclipse-tractusx

Reply:
adapted your proposals to make it clear where we create things, see https://github.com/eclipse-tractusx/sig-infra/pull/10/commits/54e87d6f9a8ee097475f50af445802f520002e39